ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/map-info/index.tsx

```diff
@@ -9,12 +9,16 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
+export { default } from './map-info'
 
-export default function leftPad(numToPad: number, size: number) {
```

Comment:
Why is github saying you are moving the `leftPad` utility? Did you accidentally delete the folder?

Reply:
It was specifically formatted for the display of coordinates for the map info component, and only used there. I didn't end up using the utility, so I deleted it

Reply:
I used the padStart function instead from the es2017 string lib https://github.com/codice/ddf/pull/4458/files/ad1277b5c44481b76de5ecc04b6765359aa63050#diff-84cd943908600452d8579e85ab19032dR57. These string padding functions should probably used from now on over custom padding functions due to their ease of use